src/client/client_context.h

```diff
@@ -72,6 +72,7 @@ class ClientContext {
 
   std::shared_ptr<ClientDestination> CreateNewLocalDestination(
       bool is_public = false,
+      // TODO(anonimal): we don't need DSA-SHA1 for default
```

Comment:
Fair enough, we should make all signature types configurable, eventually though.